resources/views/profile.blade.php

```diff
@@ -0,0 +1,56 @@
+@extends('layouts.frontend.app')
+@section('title', 'Update Proflie')
+@push('css')
+<link href="{{ asset('css/bootstrap.css') }}" rel="stylesheet">
+<link href="{{ asset('css/sb-admin-2.css') }}" rel="stylesheet">
+@endpush
+
+@section('content')
+<div class="container">
+    <h1>{{ trans('Edit profile') }}</h1>
+    <hr>
+    <div class="row">
+        <!-- left column -->
+        <div class="col-md-3">
+            <div class="text-center">
+                <form class="form-horizontal" enctype="multipart/form-data" method="POST"
+                    action="{{ route('profile.update') }}">
+                    @csrf
+                    <img id="img" src="{{ asset('storage/avatar/'. $user->image) }}" class="avatar img-circle" alt="avatar">
+                    <h6>{{ trans('upload_image') }}</h6>
+                    <input type="file" name="image" id="file" class="form-control">
+            </div>
```

Comment:
```suggestion
            <div class="text-center">
                <form class="form-horizontal" enctype="multipart/form-data" method="POST"
                    action="{{ route('profile.update') }}">
                    @csrf
                    <img id="img" src="{{ asset('storage/avatar/'. $user->image) }}" class="avatar img-circle" alt="avatar">
                    <h6>{{ trans('upload_image') }}</h6>
                    <input type="file" name="image" id="file" class="form-control">
                </form>
            </div>
```
Đóng thẻ form vào chứ a.